File: Models/Player.cs
public class Player
{
    public Player(int id, string firstName, string lastName)
    {
        Id = id;
        FirstName = firstName;
        LastName = lastName;
    }

    public Player(int id)
    {
        Id = id;
    }

    public int Id { get; }
    public string? FirstName { get; set; }
    public string? LastName { get; set; }
}

File: Bolo/ClientApp/src/app/models/Player.ts
export interface Player {
  id: number;
  firstName: string;
  lastName: string;
  score: number;
  games: any[];
  wins: number;
  losses: number;
  gamesPlayed: number;
  winPercentage: string;
  averageScore: number;
}

export interface GamePlayer {
  playerId: number;
  gameId: number;
  firstName: string;
  lastName: string;
  score: number;
}

interface PlayerGame {
  firstName: string;
  lastName: string;
  score?: number;
  id: number;
}
